src/hooks/useCourtContracts.js

```diff
@@ -564,44 +566,57 @@ export function useActiveBalanceOfAt(juror, termId) {
   return [activeBalance.amount, activeBalance.error]
 }
 
-export function useTotalActiveBalancePolling(termId) {
-  const jurorRegistryContract = useCourtContract(
-    CourtModuleType.JurorsRegistry,
-    jurorRegistryAbi
-  )
-  const [totalActiveBalance, setTotalActiveBalance] = useState(bigNum(-1))
+export function useTotalANTStakedPolling(timeout = 1000) {
+  const [totalANTStaked, setTotalANTStaked] = useState(bigNum(-1))
+  const [error, setError] = useState(false)
+  const { address: antAddress } = getKnownToken('ANT') || {}
+  const antContract = useContractReadOnly(antAddress, tokenAbi)
+
+  const controlledTimeout = useRef(0)
 
   useEffect(() => {
     let cancelled = false
     let timeoutId
 
-    const fetchTotalActiveBalance = () => {
+    if (getNetworkType() === 'local') {
```

Comment:
Wonder if we can leave a comment for this—do we not have the contracts on the local environment?